tests/LANraragi/Plugin/Metadata/Chaika.t

```diff
@@ -0,0 +1,97 @@
+use strict;
+use warnings;
+use utf8;
+use Data::Dumper;
+
+use Cwd qw( getcwd );
+use Mojo::JSON qw(decode_json encode_json);
+use Mojo::File;
+use Mojolicious;
+use LANraragi::Model::Config;
+
+use Test::More;
+use Test::Deep;
+use Test::MockObject;
+
+my $cwd = getcwd();
+my $SAMPLES = "$cwd/tests/samples";
+
+my $mock_log = Test::MockObject->new();
```

Comment:
I don't mind creating mock loggers to clean up STDOUT a bit, but if you're doing it everywhere, it might be cleaner to move those lines to `mocks.pl` under a dedicated sub like it's done for the Redis one. (`setup_logger_mock()` or something)  

That'd also make it easier to switch back to full logging when one has to debug the tests. 😛

Reply:
I agree.
This was another thing I wanted to get feedback from you before doing anything else.
Similar to this is the use of the `trap` function.